src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java

```diff
@@ -112,6 +115,14 @@ private ContentMappings(@Nonnull XmlProxy proxy) {
         ));
     }
 
+    private static Set<String> getAllowedOSName() {
+        return new HashSet<>(Arrays.asList(
+                "linux", "windows", "win", "mac", "macos", "macosx",
+                "mac os x", "ubuntu", "debian", "fedora", "red hat",
```

Comment:
While technically I think using a stop word with whitespace will work, you can add the individual words.

Reply:
I had a look on how the stop words were invoked and I saw it's by executing the `contains` method, so I'm not sure if having the different words would work. May I be missing something?

Reply:
Oh, nevermind then.